CHANGELOG.md

```diff
@@ -21,7 +21,7 @@ alongside drivers for other proprietary databases, with the goal to support full
 
 If interested, please email your inquiry to sqlx@launchbadge.com.
 
-The offline mode for the queries has been changed to use a separate file per `query!()` invocation,
+The offline mode for the queries (CLI flag `--merged`) has been changed to use a separate file per `query!()` invocation,
```

Comment:
Looking at the discussion, this doesn't seem like it would have been enough information to resolve the confusion. Adding the steps from the last reply in that thread would be much clearer.

Reply:
I understand. I'll try to rephrase it in relatively compact fashion. However I'd still explicitly mention `--merged`.

As a sqlx user, when I noticed that it doesn't work with `0.7.0` my first go to was CHANGELOG and `CTRL + F`. Searching for "offline" wasn't that obvious at first.